test/SlowTests/Server/Rachis/Cluster.cs

```diff
@@ -18,35 +19,31 @@ public class Cluster: ClusterTestBase
         public async Task test()
```

Comment:
I have already change it to CanCreateDatabaseWithReplicationFactorLowerThanTheClusterSize